.github/workflows/lint-and-test.yml

```diff
@@ -21,6 +20,9 @@ jobs:
       fail-fast: false
       matrix:
         py_version: [ '3.9' ]
+        include:
```

Comment:
Later we'll have more combinatons, but just one is supposed to run the coverage. `include` lets us say exactly which combinations we want.